src/lib/webpack/run-webpack.js

```diff
@@ -121,20 +127,20 @@ export function writeJsonStats(stats) {
 		});
 }
 
-const normalizeModule = m => {
+const stripBabelLoaderFromModuleNames = m => {
 	const keysToNormalize = ['identifier', 'name', 'module', 'moduleName', 'moduleIdentifier'];
 
 	keysToNormalize.forEach(key => {
 		if (key in m) {
-			m[key] = normalizeName(m[key]);
+			m[key] = stripBabelLoaderPrefix(m[key]);
 		}
 	});
 
 	if (m.reasons) {
-		m.reasons.forEach(normalizeModule);
+		m.reasons.forEach(stripBabelLoaderFromModuleNames);
 	}
 
 	return m;
 };
 
-const normalizeName = p => p.substr(p.lastIndexOf('!') + 1);
+const stripBabelLoaderPrefix = log => log.replace(/@?\s*(\.\/~\/babel-loader\/lib\?{[\s\S]*?}!)/g, '');
```

Comment:
I have a boatload more of these transforms we can add, I'll get clearance to release them shortly.